index.js

```diff
@@ -288,21 +297,27 @@ instance.prototype.actions = function(system) {
 				break;		
 
 			case 'no_signal':
-				cmd = '#Y 0,735,' + opt.colour + '';
+				cmd = '#Y 0,735,' + opt.colourID + '';
 				break;
 
 			case 'window_control':
-				cmd = '#Y 0,721,' + opt.window + '';
+				cmd = '#Y 0,721,' + opt.windowID + '';
 				break;	
 
 			case 'mute':
 				cmd = '#Y 0,743,' + opt.muteId + '';
 				break;
 
+			case 'hdcp':
+				cmd = '#Y 0,743,' + opt.hdcp + '';
+				break;
+
 			case 'command':
 				cmd = opt.custom;
 				break;
 
+			default:
```

Comment:
default code will run always, but in this case no code is entered, you can just remove it in next pull request